### PR TITLE
Fix usage of System.Out

### PR DESCRIPTION
**Pull request by ProjectHSI**
(at least) Paper complains about authors using System.Out instead of the Bukkit logger.
This commit fixes that.

### CODE REVIEW

File: src/main/java/com/stemcraft/SMUtil.java [hunk 1]
```diff
@@ -149,7 +149,7 @@ public static boolean loadChunkRadius(Location pos, int radius, boolean force) {
                         if(success) {
                             success = false;
                         }
-                        System.out.println("Failed to load and activate Chunk at X: " + (chunkX * 16) + " Z: " + (chunkZ * 16) + " in " + world.getName());
+                        STEMCraft.getInstance().getLogger().warning("Failed to load and activate Chunk at X: " + (chunkX * 16) + " Z: " + (chunkZ * 16) + " in " + world.getName());
```
Comment (nomadjimbob) [inline src/main/java/com/stemcraft/SMUtil.java:152]: Let's change this to severe. A chunk failing to load can indicate disk/memory issues and lead to other problems down the track

File: src/main/java/com/stemcraft/feature/SMChatFormat.java [hunk 2]
```diff
@@ -65,7 +65,7 @@ protected Boolean onEnable() {
                 format = replaceAll(PREFIX_PLACEHOLDER_PATTERN, format, () -> colorize(this.vaultChat.getPlayerPrefix(event.getPlayer())));
                 format = replaceAll(SUFFIX_PLACEHOLDER_PATTERN, format, () -> colorize(this.vaultChat.getPlayerSuffix(event.getPlayer())));
             } else {
-                System.out.println("vault null");
+                this.plugin.getLogger().warning("vault null");
```
Comment (nomadjimbob) [inline src/main/java/com/stemcraft/feature/SMChatFormat.java:68]: This whole else block can go. It is a debug message. We should never reach here based on line 43

**Comment by ProjectHSI** [comment]
Reviews will be committed shortly.

**Review by nomadjimbob — COMMENTED** [review]
Could you make the small changes in my comments. Also update the version in pom.xml to 0.3.2, then we will push it out

**Comment by nomadjimbob** [comment]
Thank you ProjectHSI

**Comment by ProjectHSI** [comment]
You're welcome.

**Pull request merged**